pandas/core/arrays/timedeltas.py

```diff
@@ -324,12 +325,106 @@ def _evaluate_with_timedelta_like(self, other, op):
 
     __mul__ = _wrap_tdi_op(operator.mul)
     __rmul__ = __mul__
-    __truediv__ = _wrap_tdi_op(operator.truediv)
     __floordiv__ = _wrap_tdi_op(operator.floordiv)
     __rfloordiv__ = _wrap_tdi_op(ops.rfloordiv)
 
+    def __truediv__(self, other):
+        # TODO: Decimals?
```

Comment:
Nit: I'm not sure I understand what this `TODO` means.

Reply:
will clarify both here and there.  question is whether we want to go out of our way to operate against `decimal.Decimal` objects

Reply:
no

Reply:
Yep, agreed